src/atoms/InputField/inputfield.scss

```diff
@@ -0,0 +1,64 @@
+@import "../../styles/variables.scss";
+
+.input-wrapper {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border: 1px solid $gray-3;
+  border-radius: 8px;
+  &.success {
+    border: 1px solid $success;
+  }
+  &.warning {
+    border: 1px solid $warning;
+  }
+  &.error {
+    border: 1px solid $error;
+  }
+  svg {
+    color: $gray-4;
+    font-size: 2rem;
+    line-height: 2rem;
+    &:first-child {
+      margin-left: 1.2rem;
+    }
+    &:last-child {
+      margin-right: 1.2rem;
+    }
+  }
+  .input {
+    width: 100%;
+    font-family: Lato;
+    font-style: normal;
+    font-weight: normal;
+    padding: 8px 12px;
+    //hack but fixes chrome default styling
+    &[type="date"],
+    &[type="time"] {
+      padding: 7px 12px;
+    }
+    font-size: 1.6rem;
+    line-height: 2.4rem;
+    color: $gray-4;
+    outline-width: 0;
+    border: unset;
+    border-radius: 0.8rem;
+  }
+
+  &.hover {
+    border: 1px solid $brand-blue-4;
+    box-shadow: 0px 0px 4px rgba(0, 109, 199, 0.4);
```

Comment:
this is the only place we're using `rgb` over a hex code can we get the hex codes for these colors off of figma? With an 8-digit hex code, the last two digits are the "alpha" channel (the a)